jwst/resample/resample.py

```diff
@@ -290,7 +290,7 @@ def resample_group(self, input_models, indices):
                 if isinstance(img, datamodels.SlitModel):
                     # must call this explicitly to populate area extension
                     # although the existence of this extension may not be necessary
-                    img.area = img.area 
+                    img.area = img.area
```

Comment:
what does this line do? does it break anything to remove it?

Reply:
I have no idea. It is a strange line but in Python, sometimes things are magical. Seems to be related to SlitModel and also used (set) in the `photom` step (multi-slit). Maybe @melanieclarke knows more.